pkg/capr/planner/etcdrestore.go

```diff
@@ -396,6 +402,23 @@ func generateManifestRemovalInstruction(runtime string, entry *planEntry) (bool,
 	}
 }
 
+// generateKubeProxyRemovalInstruction generates a rm -rf command for the kube-proxy pod-manifest.
+// This was created based on this issue: https://github.com/rancher/rancher/issues/42895
+// Despite not fixing all problems after deleting it the "waiting for calico" issue got fixed.
+func generateKubeProxyRemovalInstruction(runtime string, entry *planEntry) (bool, plan.OneTimeInstruction) {
+	if entry == nil || runtime == "" {
```

Comment:
This file won't exist on K3s and the step can be skipped. 
```suggestion
	if entry == nil || runtime != "rke2" {
```

Reply:
Changed to use rke2 but i used the capr constant instead.